modules/cli-administrator-maintenance.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/administrator-cli-commands.adoc
+
+[id='cli-maintenance-commands-{context}']
+= Maintenance CLI commands
+
+== migrate
+
+Migrate resources on the cluster.
```

Comment:
This is likely due to my ignorance with OpenShift, but can we provide any additional details on what the migration is doing? Is it migrating to a new version?

Reply:
@huffmanca I have to say that I am a bit unclear on this myself. Looking at the subcommands it seems to do something different depending (storage, image-references, etc.).

@soltysh Can you give us a more specific description about what the migrate command does?

Reply:
In the pre-4.0 world this was a required step during every upgrade to run migration to ensure we store data using the required new versions of resources (whenever they were changing versions they were stored in). In the new 4.0 world the commands should be used less frequently, since majority of those actions should be done automatically by the system. So stating that it's migrating to new version is somewhat correct (`oc adm migrate storage` is just for that), but not all commands are doing that. Other such as `oc adm migrate image-refs` is to allow changing image references when migrating images between repositories. `oc adm migrate legacy-hpa` (hopefully not needed) is meant to migrate data from the old format to the new one. Hopefully we'll deprecate these commands soon-ish :wink:

Reply:
Cool thanks for the explanation @soltysh! To give a little more detail then, we'll update it to:

"Migrate resources on the cluster to a new version or format depending on the subcommand used."

Reply:
sgtm